CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # CHANGE LOG
+## [PRERELEASE]
```

Comment:
how do you determine if changes are labeled with prerelease/version number?

Reply:
We don't know what version number anything is, until we release.  This is because we might add a new ticket that's a minor version tomorrow, etc.  So I don't like pre-emptively incrementing release versions.  

When we release, we currently bump all of the versions manually (in the changelog, package.json) and make a release PR, like this:  https://github.com/NYPL/sfr-bookfinder-front-end/pull/269 

This is one of those things that I'm pretty certain we should be able to automate with a script - like, create a release to prod PR where we can pass in the intended version number and it automatically merges a "bump" commit on dev.  However, I haven't looked too deeply into it.